snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: bittorrent-tracker
+version: master
+summary: Simple, robust, BitTorrent tracker
+description: |
+  Node.js implementation of a BitTorrent tracker, client and server.
+
+apps:
+  bittorrent-tracker:
+    command: bittorrent-tracker
+    plugs: [network, network-bind]
+
+grade: devel
+confinement: strict
+
+parts:
```

Comment:
it is internal to your project, not really used by anything else. So we can actually put whatever name we want. I've updated it to bittorrent-tracker.